public/app/features/trails/DataTrailCard.tsx

```diff
@@ -50,68 +60,140 @@ export function DataTrailCard(props: Props) {
           </div>
-          {createdAt && (
-            <i className={styles.secondary}>
-              <b>Created:</b> {dateTimeFormat(createdAt, { format: 'LL' })}
-            </i>
+          <div className={styles.primaryFont}>{dsValue && getDataSourceName(dsValue)}</div>
+        </div>
+        <div className={styles.deleteButton}>
+          {onDelete && (
+            <Card.SecondaryActions>
+              <IconButton
+                key="delete"
+                name="trash-alt"
+                className={styles.secondary}
+                tooltip="Remove bookmark"
+                onClick={onDelete}
+              />
+            </Card.SecondaryActions>
           )}
-        </Stack>
-      </Card.Actions>
-      {onDelete && (
-        <Card.SecondaryActions>
-          <IconButton
-            key="delete"
-            name="trash-alt"
-            className={styles.secondary}
-            tooltip="Remove bookmark"
-            onClick={onDelete}
-          />
-        </Card.SecondaryActions>
-      )}
-    </Card>
+        </div>
+      </Card>
+      <div className={styles.date}>
+        <div className={styles.secondaryFont}>
+          <Trans i18nKey="trails.card.date-created">Date created: </Trans>
+        </div>
+        <div className={styles.primaryFont}>{createdAt && dateTimeFormat(createdAt, { format: 'YYYY-MM-DD' })}</div>
+      </div>
+    </div>
   );
 }
 
-function getStyles(theme: GrafanaTheme2) {
+export function getStyles(theme: GrafanaTheme2) {
   return {
+    metricLabel: css({
+      display: 'inline',
+      color: 'var(--text-secondary, rgba(204, 204, 220, 0.65))',
+      fontFamily: 'Inter',
+      fontSize: '14px',
+      fontStyle: 'normal',
+      fontWeight: 400,
+    }),
+    metricValue: css({
+      display: 'inline',
+      color: 'var(--text-primary, #CCCCDC)',
+      fontFamily: 'Inter',
+      fontSize: '14px',
+      fontStyle: 'normal',
+      fontWeight: 500,
+      marginLeft: '8px', // Add space between the label and the value
+      wordBreak: 'break-all',
+    }),
     tag: css({
       maxWidth: '260px',
       overflow: 'hidden',
       textOverflow: 'ellipsis',
     }),
     card: css({
-      padding: theme.spacing(1),
+      position: 'relative',
+      width: '318px',
+      padding: `12px ${theme.spacing(2)} ${theme.spacing(1)} ${theme.spacing(2)}`,
+      height: '152px',
+      alignItems: 'start',
+      marginBottom: 0,
+      borderTop: `1px solid var(--border-Weak, rgba(204, 204, 220, 0.12))`,
+      borderRight: `1px solid var(--border-Weak, rgba(204, 204, 220, 0.12))`,
+      borderLeft: `1px solid var(--border-Weak, rgba(204, 204, 220, 0.12))`,
+      borderBottom: 'none', // Remove the bottom border
+      // eslint-disable-next-line @grafana/no-border-radius-literal
+      borderRadius: '4px 4px 0 0', // Top-left and top-right corners are 4px, bottom-left and bottom-right are 0; cannot use theme.shape.radius.default because need bottom corners to be 0
```

Comment:
Maybe there is a mistake in the UI design. @catherineymgui Could you please check the design. 
Having an eslint rule for borderRadius is weird enough and having it disabled is even weirder :D
But let's note this down to somewhere to check after merging the PR. Not a blocker. But it would be nice to understand why we have that rule in the first place and how we can play with it nicely.

Reply:
Which part of the design is this? Not sure where there should be 4px border radius

Reply:
in the new card UI, this is to make the top left and top right corners of the cards rounded, while keeping the bottom left and bottom right of the cards a right angle. i'll demonstrate what i mean in the weekly today!

Reply:
Can this be accomplished with `theme.shape.radius[something]`?

Reply:
basically, the way i have the card set up is that part A is one section and part B is a separate section. i did this to implement the different background colors between parts A and B. so part A needs to have it's top left and top right corners rounded to 4px, but the bottom left & right corners need to be a right angle (0px) since that's where part A meets part B

part B will have the top left and right corners as right angles, and the bottom left/right corners as rounded 4px
![462A9AC1-A0D0-457D-ACD2-46164FA25EB2](https://github.com/user-attachments/assets/4774271a-71e9-42b5-80a7-76fa937e3080)

if that doesn't make sense i can clarify in the daily tomorrow :)

Reply:
Yeah this makes sense. I believe in my designs it's 2px corner radius instead of 4px tho (top left and right)?

For more context, the team updated all the corner radii across Grafana from 4px to 2px sometime ago.